### ISSUE TITLE
Can't use send-keys without Cucumber

### ISSUE THREAD

**Issue by pupeno** [markgandolfo/send-keys#11]
Since the steps are loaded when the library is loaded, if you don't have the cucumber gem installed, you get an error (And doesn't exist), which makes it impossible to use the library without Cucumber.

**Comment by dgm** [comment]
agreed, this isn't a capybara gem, it's a cucumber/capybara gem. :(
